enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,88 @@
-const enhancer = require('./enhancer.js');
-// test away!
+const enhancer = require("./enhancer.js");
+
+const item1 = {
+  name: "Iron Sword",
+  enhancement: 12,
+  durability: 50
+};
+
+const item2 = {
+  name: "Golden Scythe",
+  enhancement: 4,
+  durability: 70
+};
+
+const item3 = {
+  name: "Excalibur",
+  enhancement: 20,
+  durability: 100
+};
+
+test("Should return an item with the durability restored to 100.", function() {
+  expect(enhancer.repair(item1)).toEqual({
+    name: item1.name,
+    enhancement: item1.enhancement,
+    durability: 100
+  });
+  expect(enhancer.repair(item2)).toEqual({
+    name: item2.name,
+    enhancement: item2.enhancement,
+    durability: 100
+  });
+  expect(enhancer.repair(item3)).toEqual({
+    name: item3.name,
+    enhancement: item3.enhancement,
+    durability: 100
+  });
+});
```

Comment:
there's no harm in being extra explicit or in performing multiple test for the same thing, but since the only value you could simply say `expect(enhancer.repair(item3).durability).toBe(100)` and not have to perform a deep equality check on the whole object